tests/mol-css_classes/OPS/mo/mobydick.smil

```diff
@@ -0,0 +1,65 @@
+<smil xmlns="http://www.w3.org/ns/SMIL" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0">
+    <body>
+        <seq epub:textref="../mobydick.xhtml#mobyexcerpt">
+            <par id="word1">
+                <text src="../mobydick.xhtml#c01w00001"/>
+                <audio src="../audio/mobydick.mp4" clipBegin="0:00:29.268" clipEnd="0:00:29.441"/>
+            </par>
+
+            <par id="word2">
+                <text src="../mobydick.xhtml#c01w00002"/>
+                <audio src="../audio/mobydick.mp4" clipBegin="0:00:29.441" clipEnd="0:00:29.640"/>
```

Comment:
Consider making some of these have no clipEnd / no clipBegin / bad clipEnd to see whether that interacts correctly with CSS.